config/routes.rb

```diff
@@ -34,5 +34,7 @@
   get 'errors/not_found'
   get 'errors/unauthorized'
 
+  get "/.well-known/acme-challenge/:id" => "letsencrypt#authenticate_key"
```

Comment:
Oh that's cool.  I'll do that.  I will also need to prevent this ssl redirect to prevent an error.  This has to be accessible over port 80